### PR TITLE
Fix for several answers from same player

### PR DESCRIPTION
**Pull request by spioupiou**
It looks like replacing the link_to with proper simple_forms solve the issue of multiple player_inputs :) I tried to click several times on the button and the bug was gone.
I also added some validation and a flash alert if the same user tries to submit an answer twice

https://user-images.githubusercontent.com/68413600/155287794-44e44478-af67-4a3e-8c7e-0b8b56c7c0f0.mov

### CODE REVIEW

File: app/controllers/player_inputs_controller.rb [hunk 1]
```diff
@@ -11,19 +11,23 @@ def create
...
-      render_to_string(partial: "player_input", locals: { player_input: @player_input })
-    )
+    if @player_input.save
+      # send to channel
+      RoundChannel.broadcast_to(
+        @round,
+        render_to_string(partial: "player_input", locals: { player_input: @player_input })
+      )
+    else
+      flash[:alert] = "You already replied" unless @player_input.save
```
Comment (carlnoval) [inline app/controllers/player_inputs_controller.rb:29]: We already have `if @player_input.save` on line 22 so I don't think we need `unless @player_input.save` on this line.

I tried to played with this in my local but my fingers aint fast enough to trigger the flash alert.

File: app/models/player_input.rb [hunk 2]
```diff
@@ -3,5 +3,6 @@ class PlayerInput < ApplicationRecord
   belongs_to :instance
   # belongs_to :round
   # validates :type, presence: true, inclusion: { in: %w[string boolean integer] } # can be modified later
-  # validates :value, presence: true
+  validates :input_value, presence: true
+  validates :player_id, presence: true, uniqueness: { scope: :round_id }
```
Comment (carlnoval) [inline app/models/player_input.rb:7]: Good choice limiting player input to 1 per player per round.

File: app/controllers/player_inputs_controller.rb (hunk 1 above)
Comment (spioupiou) [reply app/controllers/player_inputs_controller.rb:29]: oh my god you're absolutely right! will fix it right away sorry about that!

Comment (spioupiou) [reply app/controllers/player_inputs_controller.rb]: Solved :D

**Comment by carlnoval** [comment]
Thanks, merging.

**Pull request merged**